logic.py

```diff
@@ -1,6 +1,6 @@
 ## conditons when we have list of possible values 
```

Comment:
GPT summary of [ff48b2](https://github.com/SampatSuthar/ml-demo/blob/2ce9a211365f18cfb46299b15c0e532c3e8af178/logic.py#ff48b223391a0063edf03dca03d22da628bb8158) - [786a11](https://github.com/SampatSuthar/ml-demo/blob/eff3c18ac2d2a62721066bb2d7b050cd14c87962/logic.py#786a110b31b59ea64dca97da186af1d49882f39c):
* Changed the value of the variable `state` from 'Karnataka' to 'Rajasthan'
* Added a conditional statement to check if the value of the variable `state` is one of the given values
* Added a print statement to show the petrol price in the given state